pkg/goDB/encoder/lz4/lz4_cgo.go

```diff
@@ -1,5 +1,5 @@
-//go:build cgo
-// +build cgo
+//go:build cgo && !goprobe_noliblz4
```

Comment:
Tl;DR: Because its explicit / clearer to the user.

Longer version: I wasn't happy with how (some/most) other projects use their build tags (they are just looking at their own project, seemingly forgetting that build tags are global). Take Gin for example, they have the `jsoniter` tag to trigger using that package throughout its ecosystem. All nice, but when you compile code that uses Gin you may have no idea what that build tag belongs to and what it may trigger (especially for such a case - what if a different package uses that rather common / obvious tag and you trigger some unwanted side effect or even create a conflict of interest?)
Now of course one could argue that goProbe isn't Gin and isn't going to be consumed / included the same way, but who knows? Maybe in the future the API is included in a different project without the binaries and then someone might be happy to know that the `noliblz4` tag belongs to the goProbe package...